sass/components/navbar.sass

```diff
@@ -74,6 +74,8 @@ $navbar-bottom-box-shadow-size: 0 -2px 0 0 !default
         .navbar-link
           &::after
             border-color: $color-invert
+        & > .navbar-burger
```

Comment:
You don't really need the nesting here. Just use `.navbar-burger`.

Reply:
OK, I changed it. Is it that what you meant?

Reply:
Not exactly. Put `.navbar-burger` as a sibling of `.navbar-brand`, not as a child of `.navbar-brand`.

Reply:
Ah, I see: `.navbar-burger` does not necessarily need to be a child of `.navbar-brand`. Makes sense. So, here my next try. Is that correct? Thank's for your patience.